server/src/main/java/org/candlepin/dto/api/v1/CertificateSerialDTO.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2009 - 2017 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.CandlepinDTO;
+
+import io.swagger.annotations.ApiModel;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import java.math.BigInteger;
+import java.util.Date;
+
+
+
+/**
+ *
+ */
+@ApiModel(parent = TimestampedCandlepinDTO.class, description = "DTO representing a certificate serial")
+public class CertificateSerialDTO extends TimestampedCandlepinDTO<CertificateSerialDTO> {
+    public static final long serialVersionUID = 1L;
+
+    protected Long id;
+    protected BigInteger serial;
+    protected Date expiration;
+    protected Boolean collected;
+    protected Boolean revoked;
+
+
+    /**
+     * Initializes a new CertificateSerialDTO instance with null values.
+     */
+    public CertificateSerialDTO() {
+        // Intentionally left empty
+    }
+
+    public Long getId() {
+        return this.id;
+    }
+
+    public CertificateSerialDTO setId(Long id) {
+        this.id = id;
+        return this;
+    }
+
+    public BigInteger getSerial() {
+        return this.serial;
+    }
+
+    public CertificateSerialDTO setSerial(BigInteger serial) {
+        this.serial = serial;
+        return this;
+    }
+
+    public Date getExpiration() {
+        return this.expiration;
+    }
+
+    public CertificateSerialDTO setExpiration(Date expiration) {
+        this.expiration = expiration;
+        return this;
+    }
+
+    public Boolean isCollected() {
+        return this.collected;
+    }
+
+    public CertificateSerialDTO setCollected(Boolean collected) {
+        this.collected = collected;
+        return this;
+    }
+
+    public Boolean isRevoked() {
+        return this.revoked;
+    }
+
+    public CertificateSerialDTO setRevoked(Boolean revoked) {
+        this.revoked = revoked;
+        return this;
+    }
+
+    /**
+     * @{inheritDocs}
+     */
+    @Override
+    public String toString() {
+        return String.format("CertificateSerialDTO []");
```

Comment:
Nah, what's there works fine; it just won't do any substitutions. However, those are placeholders that I forgot to fill in.